addOns/kotlin/kotlin.gradle.kts

```diff
@@ -0,0 +1,27 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+plugins {
+    kotlin("jvm") version "1.3.50"
+}
+
+version = "1"
+description = "Allows Kotlin to be used for ZAP scripting - templates included"
+
+zapAddOn {
+    addOnName.set("Kotlin Scripting")
```

Comment:
IMO this should follow the Groovy add-on and not only add the scripting capability but provide Kotlin library as well ("Kotlin Support", just semantics/contract). Other add-ons written in Kotlin could depend on this add-on and no longer need to bundle Kotlin libs.